data/json/recipes/armor/bespoke_armor/survivor.json

```diff
@@ -65,7 +65,13 @@
     "difficulty": 7,
     "time": "12 h",
     "autolearn": true,
-    "using": [ [ "tailoring_kevlar_fabric", 200 ], [ "tailoring_neoprene_patchwork", 30 ], [ "fastener_large", 1 ], [ "clasps", 8 ] ],
+    "using": [
+      [ "tailoring_kevlar_fabric", 200 ],
+      [ "tailoring_neoprene_patchwork", 12 ],
+      [ "fastener_large", 1 ],
+      [ "clasps", 8 ],
+      [ "tailoring_nylon_patchwork", 10 ]
```

Comment:
>Can someone explain to me where is "too many alternatives" in this recipe (h20survivor_jumpsuit)? I don't understand where I should tweak it to fix.

![image](https://github.com/CleverRaven/Cataclysm-DDA/assets/99621099/1b92d6ef-7367-4a20-9861-0001ac13798a)

Reply:
So is there any way to actually simplify this recipe? Last time i tried to replace fastener_large with more clasps, but it didn't changed the situation
Hmm, maybe i could split it into "neoprene wetsuit", and then make Kevlar wetsuit use this neoprene wetsuit? 🤔

Reply:
```
  {
    "result": "h20survivor_jumpsuit",
    "type": "recipe",
    "activity_level": "LIGHT_EXERCISE",
    "category": "CC_*",
    "subcategory": "CSC_*_NESTED",
    "skill_used": "tailor",
    "difficulty": 7,
    "time": "12 h",
    "autolearn": true,
    "using": [ [ "fastener_large", 1 ], [ "clasps", 8 ] ],
    "qualities": [ { "id": "SEW", "level": 2 }, { "id": "FABRIC_CUT", "level": 2 }, { "id": "SEW_CURVED", "level": 1 } ],
    "proficiencies": [
      { "proficiency": "prof_closures" },
      { "proficiency": "prof_closures_waterproofing" },
      { "proficiency": "prof_polymerworking" },
      { "proficiency": "prof_elastics" }
    ],
    "components": [
      [ [ "filament_durable", 1310, "LIST" ] ],
      [ [ "sheet_kevlar", 200 ], [ "sheet_kevlar_patchwork", 200 ] ],
      [ [ "sheet_neoprene", 12 ], [ "sheet_neoprene_patchwork", 12 ] ],
      [ [ "sheet_nylon", 10 ], [ "sheet_nylon_patchwork", 10 ] ],
      [ [ "wetsuit", 1 ] ],
      [ [ "rubber_cement", 3 ] ]
    ]
  }
```
![image](https://github.com/CleverRaven/Cataclysm-DDA/assets/99621099/70afd582-e5c1-4e2a-89fc-28bd967ac033)

Reply:
My concern is that getting rid from the requirements means this recipe should be updated manually every time if and when someone decide to tweak this material - using req makes it "i just tweak this group and all recipes that uses it would be updated automatically"

Reply:
i tweaked it as Maleclypse proposed, to replace non-waterproof clasps with something else